walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```

Comment:
In the future, make sure that you don't edit any unrelated files (like this one, which is just an example), but for this time I'll let it pass.

Reply:
Thanks! I know this wasn't the best thing to do, but I don't know what happened to the other commit, so I decided to make this little change just so I was able to commit again and open the pull request.